docker-compose.yml

```diff
@@ -2,13 +2,24 @@
 version: "3.6"
 
 services:
+  db:
+    image: "postgres:13"
```

Comment:
We've got `postgres:11` in the GH Action config: https://github.com/opensafely-core/job-server/pull/1167/commits/ab3d40af50bdf1b440bd5f66f65e9c81c1cd90b1#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R109

Reply:
Shoot, thanks!  This is the correct one, the version in the dokku plugin (which we're not using now) is 11, but DO is 13.